.circleci/config.yml

```diff
@@ -44,6 +44,44 @@ jobs:
       - store_artifacts:
           path: build/reports/tests
 
+  publish_data:
+    executor: hmpps/java
+    docker:
+      - image: cimg/openjdk:11.0
```

Comment:
what's this image for?

Reply:
I needed a `java` executable to run the `schemaspy.jar`